src/Modules/Content/Dnn.PersonaBar.Recyclebin/Components/Prompt/Commands/PurgeUser.cs

```diff
@@ -28,7 +28,7 @@ public override ConsoleResultModel Run()
             var userInfo = UserController.Instance.GetUser(PortalId, UserId);
             if (userInfo == null)
                 return new ConsoleErrorResultModel(string.Format(LocalizeString("UserNotFound"), UserId));
-            if (!userInfo.IsDeleted)
+            if (userInfo.IsDeleted)
```

Comment:
only purge soft deleted user, should not change here

Reply:
What's the correct logic? As it was before, only soft-deleted users will be purged. With this new logic, if a user has been softly deleted (for some reason) I don't purge it.